This fixes build errors under recent Rust.

Taken from void linux: https://github.com/void-linux/void-packages/commit/5fb88cfb0a16ceb873d41560ccd8fd863ced9279

--- a/servo/components/style_traits/lib.rs
+++ b/servo/components/style_traits/lib.rs
@@ -9,7 +9,7 @@
 #![crate_name = "style_traits"]
 #![crate_type = "rlib"]
 
-#![deny(unsafe_code, missing_docs)]
+#![deny(unsafe_code)]
 
 extern crate app_units;
 #[macro_use] extern crate bitflags;
--- a/servo/components/style/lib.rs
+++ b/servo/components/style/lib.rs
@@ -23,8 +23,6 @@
 //! [cssparser]: ../cssparser/index.html
 //! [selectors]: ../selectors/index.html
 
-#![deny(missing_docs)]
-
 extern crate app_units;
 extern crate arrayvec;
 extern crate atomic_refcell;
